types/passport-local/passport-local-tests.ts

```diff
@@ -42,7 +42,7 @@ passport.use(new local.Strategy((username: any, password: any, done: any) => {
         }
 
         if (!user.verifyPassword(password)) {
-            return done(null, false);
+            return done(null, false, { message: 'something wrong', errorCode: 1 });
```

Comment:
This used to be allowed by the old type, no?  Is it possible to specify info/options without a message?

Reply:
yes, the type of info is not specified in the original docs. you can pass anything you want.

Reply:
Perhaps `message` should just be optional in the existing type?  It seems like it conveys something useful about the expected value.